File: PerformanceTesting/ResultData.cs
using System;
namespace PerformanceTesting
{
	public class ResultData
	{
		private GLCubeResults _glResults = new GLCubeResults ();
		
		public ResultData ()
		{
		}
		
		
		public GLCubeResults GLCubeResults {
			get 
			{
				return _glResults;
			}
		}
		
		private static ResultData _GResults = new ResultData ();
		
		public static ResultData Results
		{
			get 
			{
				return _GResults;
			}
		}
	}
}

File: PerformanceAppProjects/CPerformanceTesting/CPerformanceTesting/ResultData.m
//
//  ResultData.m
//  CPerformanceTesting
//
//  Created by William Holmes on 6/24/12.
//  Copyright (c) 2012 Slap Holmes Productions. All rights reserved.
//

#import "ResultData.h"

@implementation ResultData

@synthesize glCubeResults;

-(id)init
{
    self = [super init];
    if (self)
    {
        self.glCubeResults = [[GLCubeResults alloc]init];
    }
    
    return self;
}

+(ResultData*) results
{
    static ResultData* sResults = nil;
    
    if (!sResults)
        sResults = [[ResultData alloc]init];
    
    return sResults;
}

@end
